Update 404 PageNotFound Style

mikebrandin opened the pull request:
This PR updates the 404 page to have a more consistent style with the rest of the website. The changes have been tested both locally and deployed on CloudFront. Builds off of #59.

You can see these changes seen [here](https://ddyr59f88ghkk.cloudfront.net/@#@##@).

As you mentioned, I have rebased and consolidated my git history and removed unnecessary changes for this PR.

ophilli commented on site/visitor-console/src/pages/PageNotFound.tsx:
```diff
@@ -1,24 +1,24 @@
+          </div>;
+    }
+}
+export default NotFoundPage;
```
This is nitpicky, but can you `git mv site/visitor-console/src/pages/PageNotFound.tsx site/visitor-console/src/pages/NotFoundPage.tsx` so that the filepath matches the component name? Code is write-once, read-many, and little things like this will hopefully make it easier for future devs to connect the dots quicker.

mikebrandin replied:
I have made those changes in my latest commit.

ophilli commented:
Do you still want to merge this? If so you need to rebase & rebuild the site.

mikebrandin commented:
Yes, I still would like to merge this PR because the current 404 page does not match the rest of the site. I have rebased the branch, renamed the files, and updated the build folder. Please let me know if there are any other issues with this PR preventing it from being merged.

dggorsen reviewed (approved):
Can you just rebase this for me before I merge? Feel free to ping me on discord when its been rebased

mikebrandin committed:
Updated build and Filenames/Imports

mikebrandin commented:
This PR resolves issue #19
